stories/button/index.stories.tsx

```diff
@@ -108,33 +109,46 @@ import { COLORS } from 'radiance-ui/lib/constants';
-| ----------- | -------- | -------- | --------------- | ---------------------------------------------------------------------------------------------------------------------------- |
-| buttonType  | string   | no       | primary         | Determines the button's main style theme. Must be one of `primary`, `secondary`, `tertiary`, `quaternary`.                   |
-| buttonColor | string   | no       | primary         | Determines the color of the button . Must be one of the predefined colors (e.g. `primary`, `statusRed`, `statusGreen`, etc). |
-| children    | node     | yes      | -               | node to be rendered inside the button. Recommended to be the button text                                                     |
-| disabled    | bool     | no       | false           | when disabled, click listener will not be called and the UI will look disabled                                               |
-| icon        | node     | no       | null            | icon to render in the button. Recommended to use one of Radiance's icons                                                     |
-| isLoading   | bool     | no       | false           | renders loading state and prevents click listener from being called                                                          |
-| onClick     | func     | no       | () => undefined | callback function called on click of the button                                                                              |
-| textColor   | string   | no       | ''              | color (as a string) that will override existing text, icon, and loading colors for the button (except when disabled is true) |
-| isFullWidth | bool     | no       | false           | adjusts the default button styles so that the button takes the full width of the container. Intended for mobile-use only.    |
-
-### Notes
-
-Buttons can be used as a main call-to-action (CTA). Try to avoid using
-buttons of the same `buttonType` next to each other since we want to
-guide the user towards one option.
-
-`<Button.Container>` can be used to provide spacing between multiple
-buttons and behavior on various screen sizes. See the dedicated
-Readme/Story for more details.
-
-We should generally try to use the default button color when possible. Only for special cases should we need to use a different button color.
+export const WithControls = () => (
+  <Button.Container>
+    <Button
+      buttonType={select(
+        'buttonType',
+        ['primary', 'secondary', 'tertiary', 'quaternary'],
+        'primary',
+      )}
+      buttonColor={select('buttonColor', COLORS, COLORS.primary)}
+      isLoading={boolean('isLoading', false)}
+      disabled={boolean('disabled', false)}
+      onClick={action('button clicked')}
+      textColor={text('textColor', '')}
+    >
+      {text('children', 'Click me!')}
+    </Button>
+  </Button.Container>
+);
+
+export default {
+  title: 'Components/Button/Button',
```

Comment:
In order for us to nest buttons the way we currently do, we needed to add a base `Components` root node, since otherwise the `Buttons` would be separated from everything else. 

Since this was required, I also decided to split up the `constants` and the `icons` usages, since they're not `components`. 

Before:

<img width="156" alt="Screen Shot 2020-10-11 at 8 26 42 PM" src="https://user-images.githubusercontent.com/13544620/95702164-71942680-0c00-11eb-8ad2-dae336068deb.png">

After (not a 1:1 screenshot, since it takes up more space now):

<img width="142" alt="Screen Shot 2020-10-11 at 8 30 40 PM" src="https://user-images.githubusercontent.com/13544620/95702267-b029e100-0c00-11eb-9fc6-7dd689ec2651.png">